File: BuilderDesignPattern/Email_Example/Driver.java
package BuilderDesignPattern.Email_Example;

import BuilderDesignPattern.Email_Example.Director.Director;
import BuilderDesignPattern.Email_Example.ProductTypes.EmailPriorityType;
import BuilderDesignPattern.Email_Example.Products.Email;

public class Driver {
	public static void main(String[] agrs) {
		Director director = new Director();
		Email outOfOffice = director.outOfOffice("latestSendersEmail@automated.com");

		Email email1 = new Email.Builder("kartikraut@gmail.com", "Meeting Details for 3rd March 2026 as Follows:")
							   .setBody("Hi Kartik,\nHope tou are doing well. These are teh details for the StackHolders meeting details in below:\n...\nI have also attached the PRD.pdf to this email for your reference. Thank you.\n\nKind Regards\nAaditya Kasbekar.")
							   .addAttachment("/ImaportantDocuments/Projects/Abhi Corona Lega/PRD.pdf")
							   .addCCRecipient("sudhanvapaturkar@outlook.com", "akshaychanvan@yahoo.com")
							   .addBCCRecipients("sakshiwadaskar@gmail.com")
							   .changePriority(EmailPriorityType.HIGH)
							   .build();

		System.out.println(outOfOffice);
		System.out.println(email1);
	}
}

File: BuilderDesignPattern/Email_Example_Python/Driver.py
from BuilderDesignPattern.Email_Example_Python.Builders.EmailBuilder import EmailBuilder
from BuilderDesignPattern.Email_Example_Python.Director.Director import Director
from BuilderDesignPattern.Email_Example_Python.ProductTypes.EmailPriorityTypes import EmailPriorityTypes
from BuilderDesignPattern.Email_Example_Python.Products.Email import Email


class Driver:

	@staticmethod
	def main():
		director: Director = Director()
		builder: EmailBuilder = EmailBuilder("sudhanvapaturkar49@gmail.com", "Hi testing the new method.")

		email1: Email = director.outOfOfficeEmail("akshaychavan30@gmail.com")
		email2: Email = (builder.addCCRecipients("akshaykasbekar34@gmail.com")
						 .addBCCRecipients("sakshiwadaskar28@gmail.com")
						 .addAttachment("/importantDocuments/veyImportantFile")
						 .updateEmailPriority(EmailPriorityTypes.HIGH)
						 .build())

		print(email1)
		print(email2)


if __name__ == '__main__':
	Driver.main()
